### PR TITLE
backend: Entity change side effects job names should follow a patter

### PR DESCRIPTION
**Pull request by ginvaell**

### CODE REVIEW

File: src/backend.md [hunk 1]
```diff
@@ -336,13 +336,25 @@ For serverless projects - method 3 is preferred, but not always. When you have a
...
+
+   // good
+
+   async function createUser(attributes: UserAttributes) {
+     const user = await User.create(attributes);
+     await enqueueHandleUserCreated(user);
+     return user;
+   }
+
+   export async function handleUserCreated({ userId }: { userId: number }) {
```
Comment (divineforest) [inline src/backend.md:355]:
```suggestion
   export async function handleUserCreatedJob({ userId }: { userId: number }) {
```

File: src/backend.md [hunk 2]
```diff
@@ -425,15 +437,15 @@ For serverless projects - method 3 is preferred, but not always. When you have a
    ```typescript
    // bad
 
-   export async function handleUserCreatedJob({ userId }: { userId: number }) {
+   export async function handleUserCreated({ userId }: { userId: number }) {
```
Comment (divineforest) [inline src/backend.md:440]: please, revert

File: src/backend.md [hunk 3]
```diff
@@ -425,15 +437,15 @@ For serverless projects - method 3 is preferred, but not always. When you have a
...
      await updateBalance();
      await sendEmail();
      await sendAnalytics();
    }
 
    // good
 
-   export async function handleUserCreatedJob({ userId }: { userId: number }) {
+   export async function handleUserCreated({ userId }: { userId: number }) {
```
Comment (divineforest) [inline src/backend.md:448]: and here
```suggestion
   export async function handleUserCreatedJob({ userId }: { userId: number }) {
```

File: src/backend.md (hunk 2 above)
Comment (divineforest) [inline src/backend.md:440]:
```suggestion
   export async function handleUserCreatedJob({ userId }: { userId: number }) {
```

File: src/backend.md (hunk 1 above)
Comment (divineforest) [reply src/backend.md:355]: Jobs, principle 3:

> A job function should have the `Job` suffix, for example `syncTransactionsJob()`.

File: src/backend.md [hunk 4]
```diff
@@ -336,13 +336,25 @@ For serverless projects - method 3 is preferred, but not always. When you have a
      return user;
    }
 
-   // good
+   // also bad
```
Comment (dmlyapin) [inline src/backend.md:339]: Let's incorporate this point somehow, to make clear why is it "also bad":

> This does not look like a good example still. I would suggest smth like handleUserCreatedJob here. So we get read of direct side effects in create/update fns and move everything that is not the actual create/update logic out of this functions. The idea is that adding another side effect should never change the original fn

Comment (ginvaell) [reply src/backend.md:341]: done

File: src/backend.md (hunk 1 above)
Comment (ginvaell) [reply src/backend.md]: done

File: src/backend.md (hunk 2 above)
Comment (ginvaell) [reply src/backend.md]: done

File: src/backend.md (hunk 3 above)
Comment (ginvaell) [reply src/backend.md]: done

**Review by dmlyapin — APPROVED** [review]

**Pull request merged**